.dockerignore

```diff
@@ -11,5 +11,3 @@ testing/
 .git/
 .vscode/
 *.md
-go.sum
-go.mod
```

Comment:
If we ignore the go.mod/sum files then its not possible to copy these files when building the docker image as it will hide them. This I had to figure out the hard way, basically after trying everything else until I took a look at this .dockerignore file and tried commenting this out...

Reply:
@fridrik01 can we keep them ignored and just explicitly copying them over as we used to do in Dockerfile?

Reply:
No we can't keep them ignored as then we wont be able to copy them inside the `Dockerfile` as they are kept hidden from it (I probably spent good 1-2 hours debugging that)